Documentation/project-docs/developer-guide.md

```diff
@@ -197,49 +198,21 @@ There may be multiple projects in some directories so you may need to specify th
 
-#### What is TestTFM and what possible values can it have
-
-`TestTFM` is the Framework that we will use to run your tests on. The same test assembly can be used to run tests on two different frameworks, for example, AssemblyWithMyTests.dll can be used to run your tests in TFM `A` and `B` as long as frameworks `A` and `B` provide/support the NetStandard surface area needed for the test assembly to compile. For this reason, when you write your tests, you might want to run them against different frameworks, and the next section will point out how to do that.
-
-Some of the possible values for `TestTFM` are:
-
-- _**`netcoreapp1.1` or `netcoreapp1.0`**_
-NetStandard implementations that run in CoreCLR.
-
-- _**`netcore50` or `uap101aot`**_
-NetStandard implementations for UWP.
-
-- _**`net46` or `net462` or `net463`**_
-NetStandard implementations for Desktop.
-
-#### Running tests in a different TFM
-
-Each test project corresponds to a test .builds file. There are some tests that might be OS-specific, or might be testing an API that is available only on some TFMs, so the tests.builds specifies the valid configurations. By default, we will build all of these different configurations always, but we will only execute the tests on one TFM. By default, our `TestTFM` is set to `netcoreapp1.1` [here](https://github.com/dotnet/corefx/blob/80ab4804aeb7bed16e64b988c7460c705fddd5cc/dir.props#L529) (see latest value of `DefaultTestTFM` [in dir.props](https://github.com/dotnet/corefx/blob/master/dir.props)), which means that we will run tests for the `Project`s in the .builds file that either:
-- Have `netcoreapp1.1` inside their `TestTFMs` metadata, or
-- Don't have `TestTFMs` metadata, indicating the default TFM.
-
-The rest of the configurations will still get built, but they won't be executed by default, or as part of the CI. In order to use a different TestTFM, pass in the `FilterToTestTFM` property like:
-
-```cmd
-cd src\System.Runtime\tests
-msbuild System.Runtime.Tests.builds /p:FilterToTestTFM=net462
-```
-
-The previous example will again build the System.Runtime csproj in all of its different configurations, but will only execute the configurations that have `net462` in their `TestTFMs` metadata in System.Runtime.Tests.builds.
+#### Running tests in a different target framework
 
-One more way to run tests on a specific TFM is to build the test csproj directly and set the value of `TestTFM` like:
+Each test project can potentially have multiple build configurations. There are some tests that might be OS-specific, or might be testing an API that is available only on some target frameworks, so the tests `BuildConfigurations` property specifies the valid configurations. By default we will build and run only the default build configuration which `netcoreapp`. The rest of the configurations will need to be built and ran by specifying the configuration options.
```

Comment:
or by building the .builds file right? We still have those? What's the easy way to say "build all configurations"?